src/extensions/extra/HTMLHinter/parser.js

```diff
@@ -5,8 +5,8 @@ define(function(require) {
     "use strict";
 
     // Requires an `npm install` from root
-    var slowparse = require("../../../../node_modules/slowparse/slowparse");
-    var errorMessages = require("text!../../../../node_modules/slowparse/locale/en_US.json");
+    var slowparse = require("../../../thirdparty/slowparse/slowparse");
```

Comment:
No, because slowparse isn't being loaded anywhere else first. As I understand it, a module must be `require()`'d before it can be retrieved with `getModule`